test/services/katello/pulp3/migration_test.rb

```diff
@@ -78,6 +78,54 @@ def test_file_migration
           assert_equal @cv_archive_repo.publication_href, @cv_env_repo.publication_href
         end
 
+        def test_file_migration_reset_errors_out_with_no_content_types
+          migration_service = Katello::Pulp3::Migration.new(SmartProxy.pulp_primary, :repository_types => [])
+
+          assert_raise(::Katello::Errors::Pulp3MigrationError, 'There are no Pulp 3 content types to reset') do
+            migration_service.reset
+          end
+        end
+
+        def test_file_migration_reset
+          unit = @repo.files.first
+
+          service = Katello::Pulp::Repository::File.new(@repo, SmartProxy.pulp_primary)
+          service.copy_units(@cv_archive_repo, [unit])
+          service.copy_units(@cv_env_repo, [unit])
+          #only published repos will have their published metadata used as publications
+          ForemanTasks.sync_task(Actions::Katello::Repository::MetadataGenerate, @cv_archive_repo)
+          ForemanTasks.sync_task(Actions::Katello::Repository::MetadataGenerate, @cv_env_repo)
+
+          migration_service = Katello::Pulp3::Migration.new(SmartProxy.pulp_primary, :repository_types => ['file'])
```

Comment:
Do we have repo/content types as symbols or otherwise defined strings?

Reply:
We could use `::Katello::FileUnit.content_type` here, but older tests are just using the string so I stuck with consistency.

Reply:
Understood - makes sense.